sdk/python/cap/worker.py

```
@@ -12,133 +13,287 @@
+    except BaseException as exc:
+        primary_error = exc
+        raise
+    finally:
+        await _drain_connection(connection, logger, primary_error)
+
+
+async def run_worker(
+    nats_url: str,
+    subject: str,
+    handler: Callable[[job_pb2.JobRequest], Awaitable[job_pb2.JobResult]],
+    public_keys: Dict[str, ec.EllipticCurvePublicKey] = None,
```

Ruff (0.15.21)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)